builtin/credential/ldap/backend.go

```diff
@@ -183,10 +191,11 @@ func (b *backend) Login(req *logical.Request, username string, password string)
 	}
 
 	if len(policies) == 0 {
-		return nil, logical.ErrorResponse("user is not member of any authorized group"), nil
+		resp.Data["error"] = "user is not a member of any authorized group"
```

Comment:
Okay. You have told me already that you added it both in `warnings` and `errors` intentionally. I had overlooked it.